app/controllers/api/v2/reports_controller.rb

```diff
@@ -59,6 +59,12 @@ def last
         @report = Report.authorized(:view_reports).includes(:logs => [:message, :source]).find(max_id)
```

Comment:
Sure, I skipped it as I think it's not vulnerable currently due to `my_reports` being called to create `max_id` in the line above this. No harm calling it again.